src/app/notifications/components/notificationToast.tsx

```diff
@@ -32,12 +34,18 @@ export const CloseButton = (props: CloseButtonProps): JSX.Element => {
     );
 };
 
-const fetchComponent = (notification: Notification) => {
-    return <NotificationListEntry notification={notification} showAnnoucement={true}/>;
+const NotificationEntry = (props: { notification: Notification }): JSX.Element => {
+    const dispatch = useDispatch();
+
+    React.useEffect(() => {
+        dispatch!(addNotificationAction(props.notification));
```

Comment:
I am not familiar with this dispatch! syntax here, is it any different from simply dispatch?

Reply:
just checking dispatch is a function with !